### PR TITLE
change SA name from pac-manager

### PR DESCRIPTION
**Pull request by ramessesii2**
##### This PR aims at changing the name of ServiceAccount from `pac-manager` to `pipeline-service-manager`. Since, this SA is used for generating right credentials for accessing services (compute and kcp-workspace), `pac-manager` name doesn't seem be the appropriate name with respect to its function. 

Signed-off-by: Satyam Bhardwaj <sabhardw@redhat.com>

### CODE REVIEW

File: developer/ckcp/openshift_dev_setup.sh [hunk 1]
```diff
@@ -345,7 +345,7 @@ patches:
     --kcp-org "$kcp_org" \
     --kcp-workspace "$kcp_workspace" \
     --work-dir "$WORK_DIR" \
-    --kustomization "$GIT_URL/operator/gitops/kcp/pac-manager?ref=$GIT_REF" |
+    --kustomization "$GIT_URL/operator/gitops/kcp/pln-svc-manager?ref=$GIT_REF" |
```
Comment (Roming22) [inline developer/ckcp/openshift_dev_setup.sh:348]: Please change `pln-svc-manager` to `pipeline-service-manager`.

File: operator/gitops/compute/pln-svc-manager/kustomization.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Roming22) [inline operator/gitops/compute/pln-svc-manager/kustomization.yaml:1]: This directory will need to be renamed as well.

**Review by Roming22 — COMMENTED** [review]
My understanding is that the ckcp test cannot pass because of the renaming of a directory referenced in a kustomization.yaml. Theferore, I'm OK with the change being merged once my comments have been processed.

File: operator/gitops/compute/pln-svc-manager/kustomization.yaml (hunk 2 above)
Comment (ramessesii2) [reply operator/gitops/compute/pln-svc-manager/kustomization.yaml:1]: sure, I can change all those instances to `pipeline-service-manager`.

**Review by Roming22 — APPROVED** [review]

**Pull request merged**